libs/vkd3d/vkd3d_private.h

```diff
@@ -2229,7 +2229,7 @@ struct vkd3d_dynamic_state
 };
 
 /* ID3D12CommandList */
-typedef ID3D12GraphicsCommandList6 d3d12_command_list_iface;
+typedef ID3D12GraphicsCommandList9 d3d12_command_list_iface;
```

Comment:
d3d12_device_iface also needs to update to 10. I think that's what's causing the flurry of warnings.

Reply:
Done.

Reply:
There are missing conversions in e.g. d3d12_device_query_interface.
ID3D12Device9_QueryInterface(), etc.